tracing.go

```diff
@@ -252,10 +253,26 @@ func ExtractInboundSpan(ctx context.Context, call *InboundCall, headers map[stri
 
 func setPeerHostPort(span opentracing.Span, hostPort string) {
```

Comment:
I don't think we actually have this information within TChannel currently. We just have `HostPort` everywhere since that's what `net.Listen` takes, and `ln.Addr().String()` returns.

Instead of a new `peer.HostPort` tag, what about if OpenTracing had a helper to set the host/port tags based on similar logic to what you have in here?